tidb-lightning/tidb-lightning-error-resolution.md

```diff
@@ -70,7 +70,21 @@ TiDB Lightning 只能检测数据源的重复项，不能解决运行 TiDB Light
 
 ## 错误报告
 
-所有错误都会写入下游 TiDB 集群 `lightning_task_info` 数据库中的表中。在导入完成后，你可以根据数据库中记录的内容，手动进行处理。
+如果 TiDB Lightning 在运行过程中收集到报错的记录，则在退出时会同时在终端和日志中输出各个类型报错数量的统计信息。
+
+输出在终端的报错统计如下表所示：
+
+| # | ERROR TYPE | ERROR COUNT | ERROR DATA TABLE |
+| - | --- | --- | ------ |
+| 1 | Data Type | 1000 | `lightning_task_info`.`type_error_v1` |
+
+在 TiDB Lightning 的 log 文件的结尾，也会输出如下格式的错误统计信息：
+
+```shell
+[2022/03/13 05:33:57.736 +08:00] [WARN] [errormanager.go:459] ["Detect 1000 data type errors in total, please refer to table `lightning_task_info`.`type_error_v1` for more details"]
+```
```

Comment:
```suggestion
* 输出在终端的报错统计如下表所示：

    | # | ERROR TYPE | ERROR COUNT | ERROR DATA TABLE |
    | - | --- | --- | ------ |
    | 1 | Data Type | 1000 | `lightning_task_info`.`type_error_v1` |

* 输出在 TiDB Lightning 的 log 文件的结尾如下：

    ```shell
    [2022/03/13 05:33:57.736 +08:00] [WARN] [errormanager.go:459] ["Detect 1000 data type errors in total, please refer to table `lightning_task_info`.`type_error_v1` for more details"]
    ```
```